src/main/java/com/artipie/RepoConfig.java

```diff
@@ -98,6 +98,14 @@ public String type() {
         return this.string("type");
     }
 
+    /**
+     * Repo port.
+     * @return Repo port
+     */
+    public Integer port() {
```

Comment:
@Sammers21 I'd suggest to use `OptionalInt` here instead of `Integer`, to avoid nullability problems and represent absent `port` as empty value instead of `-1` as this YAML library does.